kolibri/plugins/style_guide/templates/style_guide/style_guide.html

```diff
@@ -1,3 +1,5 @@
+<link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
```

Comment:
I think we still want to allow devs to work offline.

Actually we already have [material icons](https://www.npmjs.com/package/material-design-icons) as a global NPM dependency, so the font should already be available. @christianmemije might be able to assist